blocks/identity-block/intl.json

```diff
@@ -1303,5 +1305,11 @@
 	},
 	"identity-block.sign-up-natural": {
 		"en": "Sign up"
+	},
+	"identity-block.signup-form-error.identity-already-exists": {
+		"en": "Account already exists"
+	},
+	"identity-block.terms-service-privacy-text": {
```

Comment:
@LauraPinilla Is there any reason why this is just added for "en"? Why it's different for other languages?

Reply:
Hi @malavikakoppula Unfortunately, we don't have access to the themes lokalise account.